config/slackAppKey.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I have to assume that you did not mean to check this file in? Seems like `config/slackAppKey.json` should be added to your `.gitignore`